app/assets/stylesheets/home.css.scss

```diff
@@ -10,6 +10,20 @@
   font: 0/0 a; /* Remove the gap between inline(-block) elements */
 }
 
+.new-overlay {
+  opacity: 0.9;
```

Comment:
Properties should be ordered color, font-size, opacity, padding, position, top, width